package/README.md

```
@@ -19,53 +19,48 @@
-# ****Flexible and Customizable UI****
+# **Flexible and Customizable UI**
+
 ## This can offer waveform by `wavesurfer.js`
+
 <img width="100%" src="https://user-images.githubusercontent.com/70849655/180435472-f043dbb4-54df-43e0-bc5c-67492510e817.png" alt="">
 
 ## This can offer various UI and you can also customize each component position
+
 > Full View
 > <img width="100%" src="https://user-images.githubusercontent.com/70849655/180435489-263fae23-f066-4a37-a524-58918eb40b0c.png" alt="">
 
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)